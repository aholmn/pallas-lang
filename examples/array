println([1]);
println([1, 2]);
println([1, 2, 3]);
println([1, 1*2*3/6, "Strings is also supported", true]);
println([1,3,4][3]);
