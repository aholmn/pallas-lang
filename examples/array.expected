[1]
[1, 2]
[1, 2, 3]
[1, 1, Strings is also supported, true]
RuntimeError: list index out of range